client/src/pages/add-bet/FinishBetButtons.tsx

```
@@ -75,14 +78,14 @@ export const FinishBetButtons = ({
             onClick={() => setAddStake(true)}
             className="btn btn-filled"
             children="Add Stake"
-            disabled={modifyIndex !== null}
+            disabled={modifyId !== null}
           />
           <Button
             type="button"
             onClick={handleAddToParlay}
             className="btn btn-outline"
             children="Add to parley"
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)